samples/ControlCatalog/MainWindow.xaml.cs

```diff
@@ -2,18 +2,43 @@
 using Avalonia.Controls;
 using Avalonia.Markup.Xaml;
 using System;
+using System.Threading.Tasks;
+using Avalonia.Media;
 
 namespace ControlCatalog
 {
     public class MainWindow : Window
     {
+        private int _wtf;
         public MainWindow()
         {
-            this.InitializeComponent();
+            //this.InitializeComponent();
             this.AttachDevTools();
             //Renderer.DrawDirtyRects = Renderer.DrawFps = true;
         }
 
+        async Task Wtf()
```

Comment:
Did you mean to commit these changes?

Reply:
Nope, a leftover from trying to reproduce issues with OxyPlot in controlled environment